dali/test/python/test_torch_pipeline_rnnt.py

```diff
@@ -158,16 +172,18 @@ def forward(self, inp, seq_len):
 
         # frame splicing if required
         if self.frame_splicing_stack > 1 or self.frame_splicing_subsample:
-            x = stack_subsample_frames(x, stacking=self.frame_splicing_stack, subsampling=self.frame_splicing_subsample)
+            x = stack_subsample_frames(
+                x, stacking=self.frame_splicing_stack, subsampling=self.frame_splicing_subsample)
 
         # normalize if required
         if self.normalize:
             x = self.normalize_batch(x, seq_len, normalize_type=self.normalize)
 
-        # mask to zero any values beyond seq_len in batch, pad to multiple of `pad_to` (for efficiency)
+        # mask to zero any values beyond seq_len in batch,
+        # pad to multiple of `pad_to` (for efficiency)
         max_len = x.size(-1)
-        mask = torch.arange(max_len).to(seq_len.dtype).to(x.device).expand(x.size(0),
-                                                                           max_len) >= seq_len.unsqueeze(1)
+        seq = torch.arange(max_len, dtype=seq_len.dtype, device=x.device)
+        mask = seq.expand(x.size(0), max_len) >= seq_len.unsqueeze(1)
```

Comment:
It's mostly OK, but I would have preferred not refactoring this code, since it's taken from "the reference" just to check that our suggested DALI pipeline does the same thing.

Reply:
I've run this test and it still works...